src/GraphQLParser/Visitors/SDLWriter.cs

```diff
@@ -0,0 +1,765 @@
+        where TContext : IWriteContext
+    {
+        /// <inheritdoc/>
+        public override async ValueTask VisitDocument(GraphQLDocument document, TContext context)
+        {
+            if (document.Definitions?.Count > 0)
+            {
+                for (int i = 0; i < document.Definitions.Count; ++i)
+                {
+                    await Visit(document.Definitions[i], context);
+
+                    if (i < document.Definitions.Count - 1)
+                    {
+                        await context.Writer.WriteLineAsync();
+                    }
+                }
+            }
+        }
+
+        /// <inheritdoc/>
+        public override async ValueTask VisitComment(GraphQLComment comment, TContext context)
+        {
+            int level = GetLevel(context);
+
+            bool needStartNewLine = true;
+            int length = comment.Text.Span.Length;
+            for (int i = 0; i < length; ++i)
+            {
+                if (needStartNewLine)
+                {
+                    await WriteIndent(context, level);
+                    await context.Writer.WriteAsync('#');
+                    needStartNewLine = false;
+                }
+
+                char current = comment.Text.Span[i];
+                switch (current)
+                {
+                    case '\r':
+                        break;
+
+                    case '\n':
+                        await context.Writer.WriteLineAsync();
+                        needStartNewLine = true;
+                        break;
+
+                    default:
+                        await context.Writer.WriteAsync(current);
+                        break;
+                }
+            }
+
+            await context.Writer.WriteLineAsync();
+        }
+
+        /// <inheritdoc/>
+        public override async ValueTask VisitDescription(GraphQLDescription description, TContext context)
+        {
+            int level = GetLevel(context);
+
+            await WriteIndent(context, level);
+            await context.Writer.WriteAsync("\"\"\"");
+            await context.Writer.WriteLineAsync();
+
+            bool needStartNewLine = true;
+            int length = description.Value.Span.Length;
+            for (int i = 0; i < length; ++i)
+            {
+                if (needStartNewLine)
+                {
+                    await WriteIndent(context, level);
+                    needStartNewLine = false;
+                }
+
+                char current = description.Value.Span[i];
+                switch (current)
+                {
+                    case '\r':
+                        break;
+
+                    case '\n':
+                        await context.Writer.WriteLineAsync();
+                        needStartNewLine = true;
+                        break;
+
+                    default:
+                        await context.Writer.WriteAsync(current);
```

Comment:
The sequence `"""` must be escaped.  Please add test.

Reply:
done